From 84c23210a64df2e9997816e1c3469286d09f9a36 Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Fri, 1 Feb 2019 05:00:02 +0000
Subject: [PATCH] MANTA-4102 Update libmanta to use a newer Moray client
 Reviewed by: Kody Kantor <kody.kantor@joyent.com> Approved by: Kelly
 McLaughlin <kelly.mclaughlin@joyent.com>

---
 package.json | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/package.json b/package.json
index 8d0878e..bcd5b7a 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "libmanta",
     "description": "common functions et al for manta services",
-    "version": "1.1.1",
+    "version": "1.2.0",
     "author": "Joyent (joyent.com)",
     "main": "./lib/index.js",
     "repository": {
@@ -14,7 +14,7 @@
         "bunyan": "0.22.1",
         "jsprim": "^1.3.1",
         "lru-cache": "2.3.1",
-        "moray": "3.3.0",
+        "moray": "~3.5.0",
         "once": "1.3.0",
         "restify": "2.6.3",
         "redis": "0.10.1",
-- 
2.21.0

